### PR TITLE
New release

### PR DESCRIPTION
**Pull request by RCasatta**

### CODE REVIEW

**Review by notmandatory — APPROVED** [review]
utACK 19f539dbff19f7e001cd905456ad3c87a89ef130

This `bitcoincore-rpc` version bump required to fix https://github.com/bitcoindevkit/bdk/issues/646.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- Provide errors if `rpcuser` and `rpcpassword` are provided
+
+### Changed
+
+- use bitcoin dep to 0.29.1
+
+### Fixed
+
+- fix bitcoin 0.23 on MacOS X
+- fix test flakyness
```
Comment (danielabrozzoni) [inline CHANGELOG.md:18]: nit: should be "falkiness"

**Review by danielabrozzoni — APPROVED** [review]
utACK 19f539dbff19f7e001cd905456ad3c87a89ef130

File: CHANGELOG.md (hunk 1 above)
Comment (RCasatta) [reply CHANGELOG.md:18]: fixing in following commit, thanks :)

**Pull request merged**